doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1240,14 +1240,19 @@ API Calls
 
 Once this configuration is complete, you, as a user with the *PublishDataset* permission, should be able to use the admin API call to manually submit a DatasetVersion for processing:
 
-``curl -X POST -H "X-Dataverse-key: <key>" http://localhost:8080/api/admin/submitDatasetVersionToArchive/{id}/{version}``
+``curl -X POST -H "X-Dataverse-key: <key>" http://localhost:8080/api/admin/submitDatasetVersionToArchive/{version}/{id}``
```

Comment:
This isn't right - the original is correct. Per https://guides.dataverse.org/en/latest/api/native-api.html?highlight=persistentid#get-json-representation-of-a-dataset , the ":persistentId" string is going in the {id} slot in the URL.

Reply:
Right, on the Java side we have this:

    @Path("/submitDatasetVersionToArchive/{id}/{version}")

I'll revert the doc change. Good catch, @qqmyers.